<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBroadcastDiscoveryWidget</class>
 <widget class="QWidget" name="CBroadcastDiscoveryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Broadcast Discovery</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="DiscoveryImage">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource/main.qrc">:/images/discovery1.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="DiscoveryStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="DiscoveryLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Idle.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/main.qrc"/>
 </resources>
 <connections/>
</ui>
